pgxn/neon/file_cache.c

```diff
@@ -222,7 +222,7 @@ lfc_change_limit_hook(int newval, void *extra)
 	/*
 	 * Stats collector detach shared memory, so we should not try to access shared memory here.
 	 * Parallel workers first assign default value (0), so not perform truncation in parallel workers.
-	 * Postmaster is handle SIGHUP signal, it has access to shared memory (UsedShmemSegAddr != NULL) but has no PGPROC.
+	 * The Postmaster can handle SIGHUP and it has access to shared memory (UsedShmemSegAddr != NULLL), but is has no PGPROC.
```

Comment:
Whoops I had a typo in my suggestion s/is/it at the end. My apologies :).

Reply:
Fixed